src/components/resetPassword/resetpassword.scss

```diff
@@ -6,11 +6,11 @@ body > div, body > div > div,body > div > div > div.login-form {
     padding-bottom: 10px;
     border-bottom: solid 1px rgba(230, 230, 230, 0.658);
 }
-.btn-login{
-    border-radius: 20px !important;
-    padding: 5px !important;
-    height: 35px;
-}
+// .btn-login{
```

Comment:
Remove this commented block of code

Reply:
Hi @johnwayodi,
This is not part of my original work and will need a consultation before I expunge the commented line item on the .scss file.